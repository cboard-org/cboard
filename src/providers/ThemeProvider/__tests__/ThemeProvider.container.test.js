import themeProviderContainer from '../ThemeProvider.container';
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

jest.mock('react-redux');

function Child() {
  return <p className="sub">Sub</p>;
}

describe('container group Test', () => {
  let instance;
  const props = {
    dir: '.',
    children: <Child />
  };
  beforeEach(() => {
    instance = shallow(
      <themeProviderContainer.reactComponent {...props} />
    ).instance();
  });
  it('themeProviderContainer: snapshot test', () => {
    const snapshot = renderer
      .create(<themeProviderContainer.reactComponent {...props} />)
      .toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
