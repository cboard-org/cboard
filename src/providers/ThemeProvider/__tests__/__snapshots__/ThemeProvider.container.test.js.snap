// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container group Test themeProviderContainer: snapshot test 1`] = `
<p
  className="sub"
>
  Sub
</p>
`;
